utils/src/registerReportIssueCommand.ts

```diff
@@ -3,16 +3,16 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as dayjs from 'dayjs';
-// eslint-disable-next-line import/no-internal-modules
-import * as relativeTime from 'dayjs/plugin/relativeTime';
+// import * as dayjs from 'dayjs';
```

Comment:
We can't get `dayjs` to work? 😢

Reply:
It didn't have a browser entry point in their `package.json` so I may have just assumed. I'll see if it works-- the browser entry point may only specifically be a VS Code for the Web thing,

Reply:
`browser` is a standardized property, however it's not required and I think a lot of packages haven't switched over to it yet.

Reply:
Yeah, good news, I'm pretty sure that it does work.